draft-irtf-cfrg-frost.md

```diff
@@ -773,7 +773,7 @@ parameters, to check that the signature share is valid using the following proce
 
     # Compute relation values
     l = G.ScalarBaseMult(sig_share_i)
-    r = comm_share + ((challenge * lambda_i) * PK_i)
+    r = comm_share + (challenge * lambda_i * PK_i)
```

Comment:
```suggestion
    r = comm_share + ((challenge * lambda_i) * PK_i)
```
I think this is perhaps more clear because (challenge * lambda_i) is a scalar. Maybe instead we should introduce a function explicitly for scalar multiplication, and then use it like so?

```
r = comm_share + G.ScalarMult(challenge * lambda_i, PK_i)
```

Reply:
`G.ScalarMult` would indeed be clearer- I'll open an issue for this as a nice to have, but will cut v5 with this as is.